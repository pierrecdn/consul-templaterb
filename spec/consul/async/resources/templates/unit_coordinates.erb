<%
  nodeName1 = 'consul-relay02-par.central.criteo.preprod'
  nodeName2 = 'consul06-par.central.criteo.preprod'
  node1 = coordinate.nodes.select { |c| c['Node'] == nodeName1}.first
  node2 = coordinate.nodes.select { |c| c['Node'] == nodeName2}.first
%><%= nodeName1 %> to <%= nodeName1 %>: <%= coordinate.rtt(node1['Coord'], node1['Coord']).round(8) if node1 %>s
<%= nodeName1 %> to <%= nodeName2 %>: <%= coordinate.rtt(node1['Coord'], node2['Coord']).round(8) if node1 && node2 %>s