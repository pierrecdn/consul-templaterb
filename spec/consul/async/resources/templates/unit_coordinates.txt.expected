consul-relay02-par.central.criteo.preprod to consul-relay02-par.central.criteo.preprod: 2.0e-05s
consul-relay02-par.central.criteo.preprod to consul06-par.central.criteo.preprod: 0.00496354s